.circleci/config.yml

```diff
@@ -1,8 +1,8 @@
 version: 2.1
 
 orbs:
-  hmpps: ministryofjustice/hmpps@6
-  browser-tools: circleci/browser-tools@1.1.1
+  hmpps: ministryofjustice/hmpps@7
```

Comment:
The breaking change here is to reduce the timeout of deployments from 15minutes to 5. We usually take around 2 minutes to deploy. So no further changes are required.